packages/portal-proto/src/components/Summary/CategoryTableSummary.tsx

```diff
@@ -29,12 +31,20 @@ function CategoryTableSummary<TData>({
             withinPortal={false}
           >
             <ActionIcon>
-              <InfoIcon size={16} className="text-accent" />
+              <InfoIcon
+                data-testid="button-category-table-tooltip"
+                size={16}
+                className="text-accent"
+              />
             </ActionIcon>
           </Tooltip>
         )}
       </div>
-      <VerticalTable data={data} columns={columns} />
+      <VerticalTable
+        customDataTestID={customDataTestID}
```

Comment:
Same here, will need to set up a prefix such as `text-${customDataTestID}` or something similar?